packages/core/test-helpers/core-test-helpers-kbn-server/src/create_root.ts

```diff
@@ -292,6 +307,7 @@ export function createTestServers({
           hosts: es.getHostUrls(),
           username: kibanaServerTestUser.username,
           password: kibanaServerTestUser.password,
+          ...(getFips() ? kbnSettings.elasticsearch : {}),
```

Comment:
in `is_scripting_enabled.test.ts` the `es` settings were not being copied over since I had overridden the license to be trial, so I added this just for FIPS mode so we could have the es settings propagated and the trial license

Reply:
Got it, thanks for clarifying, I think it'd worth having a comment then (my favorite comment is about comments 😆)